content/css/jenkins.css

```diff
@@ -623,7 +624,7 @@ body #sidebar ul.resources>li>a {
   line-height: 1.75rem;
   text-align: center;
   background: #999;
-  b !ackground: #335061;
+  /*b !ackground: #335061;*/
```

Comment:
Do not ever comment out code, just remove it if it's wrong. Versioning is here to help if we need to go back.